src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAOSpec.scala

```diff
@@ -45,7 +45,7 @@ class GraphWorkspaceDAOSpec extends FlatSpec with Matchers with OrientDbTestFixt
   it should "load a workspace" in withDefaultTestDatabase { dataSource =>
     dataSource.inTransaction { txn =>
       assertResult(Some(testData.workspace)) {
-        workspaceDAO.load(testData.workspace.toWorkspaceName, txn)
+        workspaceDAO.loadContext(testData.workspace.toWorkspaceName, txn) map( _.workspace )
```

Comment:
i guess these tests changes are OK, but I'd expect them to more exercise the workspaceDAO.load(...) method, loadContext is more of an internal thing we use